AlphaWallet/Settings/Views/SaveCustomRpcView.swift

```diff
@@ -154,6 +156,15 @@ class SaveCustomRpcView: UIView {
         }
     }
 
+    func unobscure(textField: TextField) {
+        guard textField != allTextFields.first else {
+            self.scrollView.setContentOffset(.zero, animated: true)
```

Comment:
Will be good to not include `self.` references unless they are necessary. It's a good indicator that it might be a strong reference in a closure.